test/performance_tests/performance_test_helper.cpp

```diff
@@ -133,6 +133,7 @@ namespace Performance_Test
         return timer.mean();
     }
 
+    /*
```

Comment:
We don't support an usage of multiline comments as it's hard to see what's really commented. Please remove the code instead.